build.gradle

```diff
@@ -38,14 +38,10 @@ subprojects {
     configurations.all {
         resolutionStrategy {
             failOnVersionConflict()
-            // Force the IDEA SDK provided version of these dependencies to avoid classloading issues.
-            force 'org.apache.httpcomponents:httpclient:4.4.1', 'com.google.guava:guava:17.0'
```

Comment:
I worry we will depend on it too much to make sure we don't add unnecessary additions. Perhaps a dependency count check is better? if total dependencies != X, then are you sure you need this dependency?